.golangci.yaml

```diff
@@ -0,0 +1,31 @@
+run:
+  timeout: 1m
+  tests: true
+
+linters:
+  enable:
+    - asciicheck
+    - forcetypeassert
+    - gocritic
+    - godot
+    - gofmt
+    - goimports
+    - misspell
+    - stylecheck
+
+issues:
+  exclude-use-default: false
+  max-issues-per-linter: 0
+  max-same-issues: 10
```

Comment:
Same here: let's use the golangci-lint defaults.

Reply:
Removed.